javascript/javascript2/week3/TheFastestPresser/fastestPresser.js

```diff
@@ -0,0 +1,128 @@
+/*==============================================================================================
+                        The fastest presser in this realm
+
+        Two users compete about who can press a key the most items within a set time!
+        A user specifies how long time the game should be, and press "Start game!"
+        When the button is pressed it is about pressing either L or S on the keyboard.
+        The user that has the highest number of keypresses after the time is up wins.
+==============================================================================================*/
+
+const userInput = document.getElementById("userInput");
+const startButton = document.getElementById("start-btn");
+const timeStatus = document.getElementById("time-checking");
+const pressS = document.getElementById("left-click-area");
+const pressL = document.getElementById("right-click-area");
+const counterS = document.getElementById("clickCounterS");
+const counterL = document.getElementById("clickCounterL");
+const resetButton = document.getElementById("reset-btn");
+
+startButton.addEventListener("click", gameStart);
+resetButton.addEventListener("click", resetStatus);
+document.addEventListener("keypress", playCount);
```

Comment:
Here probably better to use keyup or keydown event, otherwise, you can just press the button and hold it, and count will increase very fast for some players ;)